src/landusedata/regrid.py

```diff
@@ -1,21 +1,47 @@
+import numpy as np
 import xesmf as xe
+import xarray as xr
 
-def RegridConservative(ds_to_regrid, ds_regrid_target, regridder_weights, regrid_reuse):
+import sys
+
+from .utils import ImportRegridTarget
+
+class TwoStepRegridder:
+
+    def __init__(self, ds_to_regrid, regridder_steptwo_weights, intermediate_regridding_file, regrid_method="conservative"):
+        
+        self.se_regridder = make_se_regridder(regridder_steptwo_weights, regrid_method=regrid_method)
+        intermediate_ds = ImportRegridTarget(intermediate_regridding_file)
+        #print(intermediate_ds)
+        self.step_one_regridder = xe.Regridder(ds_to_regrid, intermediate_ds, regrid_method)
```

Comment:
@maritsandstad can you describe the workflow for using the intermediate regridding target?

Reply:
Thanks for the explanation @maritsandstad.  Does the interstitial regular grid just need to be a grid close in resolution to the final spectral grid resolution?  I'm not familiar with the nomeclature for the spectral grids; what's the `30` in  `ne30` refer to and what resolution in degrees is it close to?

Code comments are always welcome, but in this case I think it'd be best to add a section to the readme explaining the workflow for the spectral grid option.

Reply:
Also, it would be good to get an explanation of how to generate the map definition file and what tool you used.

Reply:
I am also not a resolution expert, but I think ne30 is around 1 degree resolution. I also don't actually know exactly how close the resolution needs to be for ESMF to be able to make a scripgrid file to define the transition. BTW I used ESMF (which is the regridding backend to xesmf) to define the grid file map definition using one of its command line prompts, I'll add the details in the README.